src/components/Header.tsx

```diff
@@ -1,5 +1,14 @@
-const Header = () => {
-  return <div>Header</div>;
+interface HeaderProps {
+  text: string;
+}
+const Header = ({ text }: HeaderProps) => {
+  return (
+    <div className="my-8 w-full px-7 text-center">
```

Comment:
the my and px can be merged into a single margin styling
```suggestion
    <div className="m-8 w-full text-center">
```